src/common/components/SupportSidebar/Mentoring.jsx

```diff
@@ -130,6 +130,19 @@ function Mentoring({
     return [];
   };
 
+  const sortByConsumptionAvailability = (allConsumables) => allConsumables.sort((a, b) => {
```

Comment:
I don't understand what you are doing here on this sort

Reply:
This is the component used for no consumption this should only be used for NON Saas students (Students that belong to at least one cohort available_as_saas=false